notebooks/dataAccess/siphon-Subset.ipynb

```diff
@@ -0,0 +1,461 @@
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "metadata = ncss.metadata\n",
+    "# print metadata\n",
+    "print(\"time span: \" + str(metadata.time_span))\n",
+    "print(\"\\naccept list: \" + str(metadata.accept_list))\n",
+    "print(\"\\nlat_lon_box: \" + str(metadata.lat_lon_box))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will use this metadata to create our subset query in the next section."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Use NCSS to query and download a subset of data\n",
+    "We can now use our NCSS client to create a query for the data we want."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Subset data at a single point\n",
+    "In this example, we'll request a subset of data containing the next 24 hours of forecast at a single point."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we create a query object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "query = ncss.query()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we populate the query to request the data we want."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "query.lonlat_point(lon=-105, lat=40) # set coordinates of point of interest.\n",
+    "now = datetime.utcnow() # get current time\n",
+    "query.time_range(now, now + timedelta(days=1)) # create time range of 24 hours\n",
+    "query.variables('Temperature_surface') # request surface temperature variable\n",
+    "query.accept('netcdf4') # return data as a netCDF4 object"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once our query is fully populated, we can request the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "point_data = ncss.get_data(query)\n",
+    "list(point_data.variables)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, let's plot our returned data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "temp = point_data.variables['Temperature_surface'][:] # get surface temperature data\n",
+    "time = point_data.variables['time'][:] # get time data\n",
+    "plt.plot(time, temp, 'k-'); # plot data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Subset data for a gridded region\n",
```

Comment:
I might actually break these two subsections into the higher level objectives and have one for point data and one for gridded data, so they obviously show up in the outline.  But it's not necessary if you don't feel like it should be that way.